packages/@azure/servicebus/data-plane/test/sessionsTests.spec.ts

```diff
@@ -309,3 +309,98 @@ describe("SessionTests - Accept a session without passing sessionId and receive
     await testComplete_batching();
   });
 });
+
+describe("SessionTests - getState and setState in Session enabled Queues/Subscriptions", function(): void {
+  afterEach(async () => {
+    await afterEachTest();
+  });
+
+  async function testGetSetState(): Promise<void> {
+    const sender = senderClient.getSender();
+    await sender.send(testMessagesWithDifferentSessionIds[0]);
+    await sender.send(testMessagesWithDifferentSessionIds[1]);
+    await delay(4000);
```

Comment:
What is this delay for?

Reply:
I didn't verify whether the test fails without `delay`.
Just followed the way we did in the precedent describe block.

Reply:
Usually we dont have delays after sending
We have them after starting a streaming receiver or if we want the lock to expire